modules/ROOT/pages/monitor-api-conformance.adoc

```diff
@@ -1,99 +1,164 @@
 = Monitoring API Conformance
 
-After you configure governance profiles for your APIs and the resulting governance data is available, you can monitor and report on your organization's API specifications' conformance success and failure.  
+After you configure governance profiles for your APIs and the resulting governance data is available, you can monitor and report on your organization's API conformance.  
 
-== Status Information on the Console
+== Test Profile Settings Using Draft Profiles
 
-You can view the latest status information in the console.
+Consider using draft profiles to test profile settings. This enables you to limit the scope of the profiles to just what you need before alerting the API owners about identified API conformance issues. 
 
-// Screenshot of the console with callouts
-
-include::_partials/api-gov-console-scap-callouts.adoc[]
-
-NOTE: Only API Governance and organization administrators can create, edit, and delete profiles.
-
-// Definition of Governance Profiles
-
-include::general::partial$api-gov-concepts.adoc[tag=governance-profiles,leveloffset=+2]
-
-// Definition of Governance Profile Status
-
-include::general::partial$api-gov-concepts.adoc[tag=governance-profile-status,leveloffset=+2]
+// Screenshot of a draft profile's page
 
-// Definition of Governed APIs
+image::api-gov-draft-profile-view.png[Screenshot of the API conformance information for a selected draft profile]
 
-include::general::partial$api-gov-concepts.adoc[tag=governed-apis,leveloffset=+2]
+[calloutlist]
+. View a summary of API conformance for the selected draft profile.
+. View the APIs targeted by the draft profile. These APIs are governed, but unless they are also targeted by an active profile, their conformance information is not revealed outside of API Governance.
```

Comment:
```suggestion
. View the APIs targeted by the draft profile.
+
These APIs are governed, but unless they are also targeted by an active profile, their conformance information is not revealed outside of API Governance.
```

Reply:
(for scannability)

Reply:
This is a tough one, since it's in a callout list. It looks a little odd with the extra space.